editor.planx.uk/src/@planx/components/Pay/Public/Pay.tsx

```diff
@@ -42,6 +43,9 @@ enum Action {
   Success,
 }
 
+export const PAY_API_ERROR_UNSUPPORTED_TEAM =
+  "GOV.UK Pay is not enabled for this local authority";
```

Comment:
Gentle reminder that Pay & Planning Constraints specifically have a dependency between their API error messages and frontend component states. API errors were recently updated here to append team name on the end and broke the frontend matching here, which I also didn't catch in previous code reviews! 

The default fronted ErrorWrapper that'll catch API errors doesn't render with a "Continue" button and fully stops a tester from proceeding, but in cases of Pay & Constraints we match on the specific "not enabled" error to show a warning while _still allowing testers to proceed_ past the Component.

Reply:
Thanks - I was hitting this yesterday afternoon whilst reviewing it and was also looking into it 🙌

Reply:
This is a good reminder that there are a few places we do this matching on error messages as opposed to statuses (I know there's a few in Save & Return / Invite to Pay also). This might be a sign that something like an enum of statuses may be a better pattern that we could consider using where it's not as simple as relying on a HTTP code.

```ts
{
  message: "Some changeable text",
  errorStatus: PayError.UnsuportedTeam
}
```

In future, it might also be nice to cover these instances with E2E tests so we get better visibility on breaking changes.